server/common/mail/templates/index.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const pupa = require('pupa');
+const base64Img = require('base64-img');
+const mjml2html = require('mjml');
+const iconPath = '/assets/shopping.png';
```

Comment:
Parse `.mjml` file as html using some _off-the-shelf_ html parser and extract path to image from markup itself instead of hardcoding it here :bowtie: